src/library/PoolAccount/index.tsx

```diff
@@ -12,7 +12,7 @@ import Identicon from 'library/Identicon';
 import { useConnect } from 'contexts/Connect';
 import { ConnectContextInterface } from 'types/connect';
 import { u8aToString, u8aUnwrapBytes } from '@polkadot/util';
-import { BatchKeys } from 'library/BatchKeys';
+import { BatchKeys } from 'library/batchKeys';
```

Comment:
You misunderstood this. I asked the variable `batchkeys` to be camel cased to `batchKeys`. Not the class / file name.